### PR TITLE
Eca/scaffold v2 - Updating Azure Scaffold to V2

### PR DESCRIPTION
**Pull request by rdendtler**
This PR is the newly updated content for the Azure scaffold representing the full re-write to version 2, which extends and builds on the existing content.

### CODE REVIEW

File: docs/cloud-adoption/appendix/azure-scaffold.md [hunk 1]
```diff
@@ -2,247 +2,331 @@
 title: Best practices for enterprises moving to Azure
 description: Describes a scaffold that enterprises can use to ensure a secure and manageable environment.
 author: rdendtler
+
 ms.date: 03/31/2017
```
Comment (bennage) [inline docs/cloud-adoption/appendix/azure-scaffold.md]: We need to update the date.

File: docs/cloud-adoption/appendix/azure-scaffold.md [hunk 2]
```diff
@@ -2,247 +2,331 @@
...
+The [Azure Security Center](https://docs.microsoft.com/en-us/azure/security-center/security-center-intro) provides a unified view of the security status of resources across your environment in addition to advanced threat protection. Azure Security Center is an open platform that enables Microsoft partners to create software that plugs into and enhance its capabilities. The baseline capabilities of Azure Security Center (free tier) provides assessment and recommendations that will enhance your security posture. Its paid tiers enable additional and valuable capabilities such as Just In Time admin access and adaptive application controls (whitelisting).
 
-This proliferation of subscriptions is no longer needed. With [role-based access control](/azure/role-based-access-control/overview), you can assign users to standard roles (such as common "reader" and "writer" types of roles). You can also define custom roles.
+> [!TIP] Azure security center is a very powerful tool that is constantly being enhanced and incorporating new capabilities you can leverage to detect threats and protect your enterprise. It is highly recommended to always enable ASC.
```
Comment (bennage) [inline docs/cloud-adoption/appendix/azure-scaffold.md]: I think that there needs to be a newline after `> [!TIP]`

File: docs/cloud-adoption/appendix/azure-scaffold.md [hunk 3]
```diff
@@ -2,247 +2,331 @@
...
+The scaffold is based on practices we have gathered from many engagements with clients of various sizes. Those clients range from small organizations developing solutions in the cloud to large multi-national enterprises and independent software vendors who are migrating workloads and developing cloud-native solutions. The enterprise scaffold is "purpose-built" to be flexible to support both traditional IT workloads and agile workloads; such as, developers creating software-as-a-service (SaaS) applications based on Azure platform capabilities.
 
-![scaffold components](./_images/components.png)
+The enterprise scaffold is intended to be the foundation of each new subscription within Azure. It enables administrators to ensure workloads meet the minimum governance requirements of an organization without preventing business groups and developers from quickly meeting their own goals. Our experience shows that this greatly speeds, rather than impedes, public cloud growth.
```
Comment (MikeWasson) [inline docs/cloud-adoption/appendix/azure-scaffold.md]: > public cloud growth

This phrase sounds like it's talking about *our* growth, maybe say adoption of the cloud?

File: docs/cloud-adoption/appendix/azure-scaffold.md [hunk 4]
```diff
@@ -2,247 +2,331 @@
...
 
 > [!NOTE]
-> Azure has grown rapidly since its introduction in 2008. This growth required Microsoft engineering teams to rethink their approach for managing and deploying services. The Azure Resource Manager model was introduced in 2014 and replaces the classic deployment model. Resource Manager enables organizations to more easily deploy, organize, and control Azure resources. Resource Manager includes parallelization when creating resources for faster deployment of complex, interdependent solutions. It also includes granular access control, and the ability to tag resources with metadata. Microsoft recommends that you create all resources through the Resource Manager model. The enterprise scaffold is explicitly designed for the Resource Manager model.
-> 
-> 
+> Microsoft has released into preview a new capability called Azure Blueprint that will enable you to package, manage, and deploy common images, templates, policies, and scripts across subscriptions and management groups. This capability is the bridge between the scaffold's purpose as reference model and deploying that model to your organization.
```
Comment (MikeWasson) [inline docs/cloud-adoption/appendix/azure-scaffold.md]: Do you have a link for Azure Blueprint?

Comment (rdendtler) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: I don't.  The content will be released next week and I was going to do an update then, also for Azure Resource Graph

File: docs/cloud-adoption/appendix/azure-scaffold.md [hunk 5]
```diff
@@ -2,247 +2,331 @@
...
 
-> [!IMPORTANT]
-> Governance is crucial to the success of Azure. This article targets the technical implementation of an enterprise scaffold but only touches on the broader process and relationships between the components. Policy governance flows from the top down and is determined by what the business wants to achieve. Naturally, the creation of a governance model for Azure includes representatives from IT, but more importantly it should have strong representation from business group leaders, and security and risk management. In the end, an enterprise scaffold is about mitigating business risk to facilitate an organization's mission and objectives.
-> 
-> 
+In real life, scaffolding is used to create the basis of a structure. The scaffold guides the general outline and provides anchor points for more permanent systems to be mounted. An enterprise scaffold is the same: a set of flexible controls and Azure capabilities that provide structure to the environment, and anchors for services built on the public cloud. It provides the builders (IT and business groups) a foundation to create and attach new services keeping speed of delivery in mind.
```
[the remark this answers is not in the document]
Comment (rdendtler) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: Good point.  Updating with "When creating a building..."

File: docs/cloud-adoption/appendix/azure-scaffold.md (hunk 3 above)
Comment (rdendtler) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: Fixed/updated.  Agreed on point.

File: docs/cloud-adoption/appendix/azure-scaffold.md (hunk 4 above)
Comment (BrianBlanchard) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: Does this mean that we need to hold on publication until the Azure Blueprint link is released? Or do we publish now & have @Petertaylor9999 update to add the link in a few days?

Comment (MikeWasson) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: I think so? Otherwise we're referencing something that doesn't exist yet.

Comment (rdendtler) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: I'm OK with waiting until the link is released. It should be any day so timing it to release with that content makes sense.

Comment (rdendtler) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: Just looked and it appears that the BluePrints documentation has been released. I'm updating the content right now.

Comment (BrianBlanchard) [reply docs/cloud-adoption/appendix/azure-scaffold.md]: Thanks @rdendtler This change avoids potential confusion & allows us to meet quality standards.

**Pull request merged**